### DISCUSSION TITLE
Is there a way to prevent Auto-DJ Stations from starting after a server reboot without music assigned to them, like before.

### DISCUSSION THREAD

**Discussion by renatopinto89** [AzuraCast/AzuraCast#5871] [Q&A]
Hi guys,
Thank you so much for this amazing project.
I´m aware that in a previous stable version of AzuraCast, Auto-DJ Stations would refuse to start if there was no music assigned to any playlists.
I know that gave some problems to some users and was removed on current versions, but is that a way to have that as a option? It´s handy in case the server restarts for some reason and saves resources.
Thank you.

**Comment by BusterNeece** [comment]
@renatopinto89 We still don't start up stations that aren't marked as having started at least once, so that should help with things. With other stations, you can temporarily disable them to reduce the resources they consume.
